cmd/codegen/generator/go/templates/modules.go

```diff
@@ -1078,11 +1092,11 @@ func (ps *parseState) declForFunc(fnType *types.Func) (*ast.FuncDecl, error) {
 // commentForFuncField returns the *ast* comment group for the given position. This
 // is needed because function args (despite being fields) don't have comments
 // associated with them, so this is a neat little hack to get them out.
-func (ps *parseState) commentForFuncField(fnDecl *ast.FuncDecl, unpackedParams []*ast.Field, i int) (*ast.CommentGroup, error) {
+func (ps *parseState) commentForFuncField(fnDecl *ast.FuncDecl, unpackedParams []*ast.Field, i int) (docComment *ast.CommentGroup, lineComment *ast.CommentGroup) {
 	pos := unpackedParams[i].Pos()
 	tokenFile := ps.fset.File(pos)
 	if tokenFile == nil {
-		return nil, fmt.Errorf("no file for function %s", fnDecl.Name.Name)
+		return nil, nil
```

Comment:
Are there situations in which this is expected to happen now? just curious

Reply:
I don't think it was really possible before either actually - since we only ever ran `commentForFuncField` on functions in the current module, and they were always parsed. Even in the context of https://github.com/dagger/dagger/pull/6185, where functions might be outside of the current package, then the `fset` is still populated for files in those other packages (though they haven't been fully parsed into a syntax tree).

So really this case wasn't actually possible to trigger, so we don't need to bother about returning an error (and even if there's a bizarre edge case, this just means that we don't return any comments for that field, which seems like the right behavior).